docs/troubleshooting.md

```diff
@@ -1,21 +1,41 @@
 ---
 layout: default
-nav_order: 7
-title: Troubleshooting
-
-nav_exclude: true
+nav_order: 4
+title: Troubleshooting a plug-in
 ---
 
 # {{ page.title }}
+{: .no_toc }
+When a plug-in does not behave as expected, you need the means to analyze the problem. In this section we will show you what tools are available to do just that.
+
+## Table of Contents
+{: .no_toc }
+1. TOC
+{:toc}
+
+## Logging
+Looking at a log file is often the most accessable way of problem analysis after an error occured. To support this, import plug-ins of any type can add entries to the regular application log of *PiWeb Auto Importer*. More details about how to do this and where to find the resulting logs are available in our dedicated [Application logs]({% link docs/plugin_fundamentals/logging.md %}) article.
+
+## Debugging
+When a missbehavior is reproducible, debugging is usually the easiest way to find the problem. An import plug-in can be debugged simply by attaching a debugger to an *PiWeb Auto Importer* process using the plug-in. Ideally a debug build of the plug-in. If you have used our project template to create the .NET project of the plug-in (see), the project already contains a launch profile to do this in a single click:
 
-<!---
-Ziele:
-- aufzeigen, wie man sehen kann, was das Problem bei der Ausführung des Plug-ins ist
+![Debug plug-in](../assets/images/troubleshooting/debugging.png "Debug plug-in")
 
-Inhalt:
-- auf Fehlermeldungen im Plug-in Management View hinweisen
-- PiWeb Log erklären
-- evtl. typische Probleme und deren Lösung auflisten
---->
+If you do not have this launch configuration, you can easily add it to your project by creating a `launchSettings.js` file in the `Properties` folder of your project:
+```json
+{
+  "profiles": {
+    "AutoImporter": {
+      "commandName": "Executable",
+      "workingDirectory": "$(ProjectDir)",
+      "executablePath": "%ProgramFiles%\\Zeiss\\PiWeb\\AutoImporter.exe",
+      "commandLineArgs": "-pluginSearchPaths $(OutDir)"
+    }
+  }
+}
+```
 
-<!-- TODO The message displayed when starting AutoImporter with pluginSearchPath when development mode is not active needs to be updated to link to the documentation (currently to the Import Sdk on GitHub). -->
\ No newline at end of file
+{: .note }
+> For this to work correctly, two conditions need to be met:
+> - *PiWeb Auto Importer* must be installed locally. The executable is expected to be found in <span class="nowrap">`%ProgramFiles%\Zeiss\PiWeb\AutoImporter.exe`</span>. If the *PiWeb Auto Importer* executable is in another path, you can update the path specified in `launchSettings.json` accordingly.
```

Comment:
Please add a line break after the first sentence of each condition.

Reply:
Done.